compiler/x/codegen/OMRInstOpCode.enum

```diff
@@ -83,3 +83,5 @@ VPROLVDRegRegMem = VPROLVDRegRegReg,
 VPROLVQRegRegMem = VPROLVQRegRegReg,
 VPRORVDRegRegMem = VPRORVDRegRegReg,
 VPRORVQRegRegMem = VPRORVQRegRegReg,
+PMOVSXBDRegMem   = PMOVSXBDRegReg,
+PMOVSXWDRegMem   = PMOVSXWDRegReg,
```

Comment:
Do the RegMem forms need tests as well?

Reply:
I think its OK without them, but I added a few tests just to be sure